static/app/components/sidebar/index.tsx

```diff
@@ -100,9 +105,14 @@ function Sidebar({location, organization}: Props) {
   // Trigger panels depending on the location hash
   useEffect(() => {
     if (location?.hash === '#welcome') {
-      togglePanel(SidebarPanelKey.OnboardingWizard);
+      activatePanel(SidebarPanelKey.OnboardingWizard);
+    } else if (
+      location?.hash === '#performance-sidequest' &&
```

Comment:
I thought what this would do is that it'll open the panel on all URLs as long as it has the `performance-sidequest` tag?